lighthouse-core/gather/driver.js

```diff
@@ -1454,6 +1458,14 @@ class Driver {
 
     await this.sendCommand('Page.enable');
   }
+
+  /**
+   * Set chrome security settings to ignore certificate errors.
+   * @return {Promise<void>}
+   */
+  async setIgnoreHttpsErrors() {
+    await this.sendCommand('Security.setIgnoreCertificateErrors', {ignore: true});
```

Comment:
I believe puppeteer uses 

```js
await client.send('Security.setOverrideCertificateErrors', {override: true});
client.on('Security.certificateError', event => this._onCertificateError(event));

_onCertificateError(event) {
    this._client.send('Security.handleCertificateError', {
      eventId: event.eventId,
      action: 'continue'
    }).catch(debugError);
}
```

but in classic Google fashion those two methods are `deprecated` and `setIgnoreCertificateErrors ` is `experimental`, soooo we're stuck between a 💎and a hard place

Reply:
Seems like lighthouse already uses some experimental methods already, like e.g. [Emulation.setCPUThrottlingRate](
https://github.com/GoogleChrome/lighthouse/blob/d76b2f5e02df7b04d861c25f723bc69514d2e863/lighthouse-core/lib/emulation.js#L136).
That being said, it looks like puppeteer is actually [waiting for it to stabilize](https://github.com/GoogleChrome/puppeteer/issues/1634). I'm fine with either, but I'd do `setIgnoreCertificateErrors`.

Reply:
Oh yeah we use plenty of experimental that's not an issue on its own, it's just rarer that there is an alternative in the "stable" protocol set we can use

Reply:
I changed to the puppeteer method, for some strange reason, this conflicts with the `redirectPass` (and only with that pass).
![Screenshot 2019-03-19 at 16 59 04](https://user-images.githubusercontent.com/2109932/54621615-53883380-4a68-11e9-9331-57c928ebf3e7.png)
Seems like internally chrome executes the `Security.handleCertificateError` just fine though.